mpi-proxy-split/p2p_drain_send_recv.cpp

```diff
@@ -76,7 +75,15 @@ registerLocalSendsAndRecvs()
   // Get a copy of MPI_COMM_WORLD
   MPI_Group group_world;
   MPI_Comm mana_comm;
-  MPI_Comm_internal_virt_group(MPI_COMM_WORLD, &group_world);
+  // MPI_Comm_group creates the same real id (and therefore, virtual id)
+  // for identical calls. If the application already has this virtual id, no
+  // extra resource is created.
+  //
+  // See mpi-proxy-split/virtual-ids.h, 214-223.
```

Comment:
Line numbers like `214-223` can change over time.  Can you identify this by the function or declaration that it relates to in mpi-proxy-split/virtual-ids.h?